modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportGWAS.pm

```diff
@@ -264,6 +266,9 @@ sub parse_input_file {
              $unit =~ s/\)//g;
              $unit =~ s/µ/micro/g;
           if ($unit =~ /decrease|increase/) {
+            $unit =~ s/ //g;
+            unidecode($unit);
```

Comment:
If the spaces are removed then in the table we will have values:
```
0.020274 SDunitincrease
0.03160937 unitdecrease
6.72 zscoreincrease
```

---
$unit = unidecode($unit);